Move framework examples higher and mention popular CMSes

fureigh opened the pull request:
This PR adds links to our Drupal, WordPress and SuiteCRM examples so those terms will be indexed for site search and otherwise made easier to find on the [Languages and Frameworks](https://cloud.gov/docs/apps/frameworks/) page of the docs.

Context: potential customers have been asking whether they can run specific CMSes on cloud.gov. Though several are represented in our `cf-example-*` repos, they're currently discoverable only via the`cf-hello-worlds` repo's README.

Minimally closes https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-6009.

fureigh committed:
Move framework examples higher and mention popular CMSes

Potential customers have been asking whether they can run specific
CMSes on cloud.gov. Though several are represented in our
`cf-example-*` repos, they're currently discoverable only via the
`cf-hello-worlds` repo's README. This commit adds links to our
Drupal, WordPress and SuiteCRM examples so those terms will be
indexed for site search and otherwise made easier to find.

https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-6009

jcscottiii commented on content/docs/apps/frameworks.md:
```diff
@@ -17,6 +17,10 @@ cloud.gov uses [buildpacks]({{< relref "docs/getting-started/concepts.md#buildpa
 
+## Examples
+
+Check out [cloud.gov's collection of Hello World applications](https://github.com/18F/cf-hello-worlds) and [Cloud Foundry's sample applications](https://github.com/cloudfoundry-samples) for examples in many different languages and frameworks, including [Drupal](https://github.com/18F/cf-ex-drupal), [WordPress](https://github.com/18F/cf-ex-wordpress) and [SuiteCRM](https://github.com/18F/cf-exampl ...
```
I wonder if we should break this down into a list of options. 1) Hello Worlds 2) CloudFoundry sample apps 3) Drupal 4) Wordpress 5) SuiteCRM. (not for or against it, just thinking) any thoughts?

brittag replied:
That makes sense to me, to give each option a little more room.

fureigh replied:
@jcscottiii I took a shot in https://github.com/18F/cg-site/pull/946/commits/b134078883e4752b6fa897996df9b35934229213. Ready for another look!

fureigh committed:
Rephrase framework section

Rework this section to highlight 1) specific examples, 2) cloud.gov
'hello world' apps, then 3) CloudFoundry 'hello world' apps, per
conversation with @jcscottiii.

https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-6009
